packages/app-mobile/components/FolderPicker.tsx

```diff
@@ -1,5 +1,4 @@
-const React = require('react');
-
+import * as React from 'react';
```

Comment:
After switching to React 19, using `const React = require('react');` in a `.ts` file causes TypeScript errors. For example,
```
root.tsx:1408:8 - error TS2786: 'MenuProvider' cannot be used as a JSX component.
  Its type 'MenuProviderStatic' is not a valid JSX element type.
    Type 'MenuProviderStatic' is not assignable to type 'new (props: any, deprecatedLegacyContext?: any) => Component<any, any, any>'.
      Property 'refs' is missing in type 'Component<MenuProviderProps, any, any>' but required in type 'Component<any, any, any>'.
```